xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/UnsignedByteArrayTest.java

```diff
@@ -41,4 +42,96 @@ public void lowerCaseOrUpperCase() {
     assertThat(UnsignedByteArray.fromHex("abcdef0123").hexValue()).isEqualTo("ABCDEF0123");
   }
 
+  @Test
+  public void simpleEmpty() {
+    assertThat(UnsignedByteArray.of(new byte[] {0}).empty()).isEqualTo(UnsignedByteArray.of(new byte[] {}));
+    assertThat(UnsignedByteArray.of(new byte[] {0, MAX_BYTE}).empty()).isEqualTo(UnsignedByteArray.of(new byte[] {}));
```

Comment:
Why do we need these? `empty()` is a static method, so it's a bit weird to call `empty()` on an instance of `UnsignedByteArray`. Also, `empty()` just creates a new empty `UnsignedByteArray`, so why create a new `UnsignedByteArray` and then call `empty()`?

Reply:
@nkramer44 I think this is now resolved with the latest commits from @mukulljangid - plz make another comment if you see something missing.